heartbeat/aws-vpc-move-ip

```diff
@@ -243,7 +243,7 @@ ec2ip_monitor() {
 				ocf_log debug "executing command: $cmd"
 				ROUTE_TO_INSTANCE="$($cmd)"
 			else
-				cmd="$OCF_RESKEY_awscli --output text ec2 describe-route-tables --route-table-ids $rtb --query RouteTables[*].Routes[?DestinationCidrBlock=='$OCF_RESKEY_ip/32'].$OCF_RESKEY_lookup_type"
+				cmd="$OCF_RESKEY_awscli $region_opt --output text ec2 describe-route-tables --route-table-ids $rtb --query RouteTables[*].Routes[?DestinationCidrBlock=='$OCF_RESKEY_ip/32'].$OCF_RESKEY_lookup_type"
```

Comment:
Thanks! I was thinking in 324f3ba that `$region_opt` wasn't necessary here because it's applied in `execute_cmd_as_role()`. However, I see now that it's inserted for a separate command in `execute_cmd_as_role()`; it still needs to be applied to `cmd`.